ppr-api/src/ppr_api/models/type_tables.py

```diff
@@ -18,17 +18,117 @@
 from .db import db
 
 
+class CountryType(db.Model):  # pylint: disable=too-few-public-methods
+    """This class defines the model for the country_type table."""
+
+    __tablename__ = 'country_types'
+
+    country_type_cd = db.Column('country_type_cd', db.String(2), primary_key=True)
+    country_desc = db.Column('country_desc', db.String(75), nullable=False)
+
+    # parent keys
+
+    # Relationships - Address
+    address = db.relationship('Address', back_populates='country_type')
+
+
+class ProvinceType(db.Model):  # pylint: disable=too-few-public-methods
+    """This class defines the model for the province_type table."""
+
+    __tablename__ = 'province_types'
+
+    province_type_cd = db.Column('province_type_cd', db.String(2), primary_key=True)
```

Comment:
why **type**?

Reply:
Do you mean the name of the file? The models in this file represent the type tables in the PPR data model.